bin/wmagent-upload-config

```diff
@@ -26,3 +26,6 @@ if __name__ == "__main__":
 
     removeWMAgentConfig(reqMgrAux, wmConfig.Agent.hostName)
     insertWMAgentConfig(reqMgrAux, wmConfig.Agent.hostName, DEFAULT_AGENT_CONFIG)
+    # for some reason, sometimes the document doesn't get created
+    print("Trying to insert it again...")
```

Comment:
Instead trying again maybe it checks document is inserted, if not insert again.
Anyway, Alan, I will create the new patch, since you have a lot things to do.